packages/controller-utils/package.json

```diff
@@ -29,6 +29,8 @@
     "test:watch": "jest --watch"
   },
   "dependencies": {
+    "@metamask/utils": "^3.3.1",
+    "@spruceid/siwe-parser": "^2.0.2",
```

Comment:
Shall we play it safe and match the version from the extension so `^1.1.3`?

Reply:
Thanks for pointing that out Matthew, that makes sense